torchbenchmark/models/BERT_pytorch/__init__.py

```diff
@@ -149,6 +149,10 @@ def __init__(self, test, device, batch_size=None, jit=False, extra_args=[]):
                                    lr=args.lr, betas=(args.adam_beta1, args.adam_beta2), weight_decay=args.adam_weight_decay,
                                    with_cuda=args.with_cuda, cuda_devices=args.cuda_devices, log_freq=args.log_freq, debug=args.debug)
 
+        if test == "eval":
+            print("optimize bert for eval mode")
```

Comment:
Can you please remove this unnecessary log?

Reply:
Hi @xuzhao9 i thought this is going to be informative, hence added intentionally.

Reply:
Sorry but our downstream customers (e.g. torchdynamo/inductor) will expect minimum logs from the model code. Therefore I believe it will be better to remove this logging code.

Reply:
sure, will remove it.

Reply:
Thanks!!